src/extension/app/plugins/publish/publish.js

```diff
@@ -35,27 +35,16 @@ export function createPublishPlugin(appStore) {
     button: {
       text: i18n(appStore.languageDict, 'publish'),
       action: async (evt) => {
-        const { siteStore, location } = appStore;
+        const { location } = appStore;
         const path = location.pathname;
         appStore.showWait();
-        let urls = [path];
-        // purge dependencies
-        if (Array.isArray(window.hlx.dependencies)) {
-          urls = urls.concat(window.hlx.dependencies);
-        }
-        const results = await Promise.all(urls.map((url) => appStore.publish(url)));
-        if (results.every((res) => res && res.ok)) {
-          // fetch and redirect to production
-          const redirectHost = siteStore.host || siteStore.outerHost;
-          const prodURL = `https://${redirectHost}${path}`;
-
-          // eslint-disable-next-line no-console
-          console.log(`redirecting to ${prodURL}`);
-
+        const res = await appStore.publish(path);
+        if (res.ok) {
+          appStore.hideWait();
           appStore.switchEnv('prod', newTab(evt));
```

Comment:
```suggestion
          const liveHost = siteStore.host || siteStore.outerHost;
          // TODO: i18n.. probably needs better wording too.
          appStore.showToast(`Successful publish: <a href='https://${liveHost}' target='_blank'>View</a>`, 'positive');
```

Reply:
This would also need to be changed to `innerHTML`

https://github.com/adobe/aem-sidekick/blob/main/src/extension/app/components/toast/toast-container.js#L60

Reply:
Let's change this in a seperate PR, after careful stakeholder consultation 😉